.dev/docker/doc/Dockerfile

```diff
@@ -11,4 +11,4 @@ RUN apt-get update \
       pandoc \
  && rm -rf /var/lib/apt/lists/*
 
-RUN pip3 install Jinja2==2.8.1 sphinx sphinxcontrib-doxylink requests grip
+RUN pip3 install Jinja2==2.8.1 sphinx sphinxcontrib-doxylink sphinx_rtd_theme requests grip
```

Comment:
@kunaltyagi it's gonna keep throwing errors until `sphinx_rtd_theme` is added to our doc image

Reply:
I added it to doc image and pushed it. Please check the latest `pointcloudlibrary/doc`

Reply:
Docker hub seems to suggest otherwise. That is the build log from the latest image.
https://hub.docker.com/layers/pointcloudlibrary/doc/latest/images/sha256-41549b35efcfca4d537b7ab81b3f663cf1bb0a7ea845890b97e9acce74c0453b?context=repo

Reply:
That happened due to the new Docker pipeline. I missed this side-effect.

Reply:
Does that mean we no longer need to manually push images?